addons/base_vat/models/res_partner.py

```diff
@@ -751,6 +752,16 @@ def check_vat_br(self, vat):
         is_cnpj_valid = stdnum.get_cc_module('br', 'cnpj').is_valid
         return is_cpf_valid(vat) or is_cnpj_valid(vat)
 
+    __check_vat_cr_re = re.compile(r'^(?:[1-9]\d{8}|\d{10}|[1-9]\d{10,11})$')
+
+    def check_vat_cr(self, vat):
+        # CÉDULA FÍSICA: 9 digits, no leading zeros or hyphens
+        # CÉDULA JURÍDICA: 10 digits, no leading zeros or hyphens
```

Comment:
the 10 digit vat numbers do allow leading zeros, so this comment is still misleading, no?